src/Form/StatisticsPluginListForm.php

```diff
@@ -107,23 +111,40 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#type' => 'table',
       '#header' => [
         'id' => $this->t('ID'),
-        'label' => $this->t('Label')
+        'label' => $this->t('Label'),
+        'operations' => $this->t('Operations')
       ],
       '#empty' => $this->t('There are no plugins yet.'),
       '#tableselect' => TRUE,
       '#default_value' => $this->config('sapi.action_handlers')->get('enabled'),
     ];
+
     // Loop through the statistics plugins.
     foreach ($this->sapi_action_handler_manager->getDefinitions() as $pluginDefinition) {
       $id = $pluginDefinition['id'];
+
       $label = $pluginDefinition['label'];
 
       $form['plugins']['action_handlers'][$id] = [
         'id' => ['#plain_text' => $id],
         'label' => ['#plain_text' => $label]
       ];
-    }
 
+      $form['plugins']['action_handlers'][$id]['operations'] = [
+        '#type' => 'operations',
+      ];
+
+      // TODO: try creating plugins
+      $instance = $this->sapi_action_handler_manager->createInstance($id);
+
+      if ($instance instanceof PluginFormInterface){
```

Comment:
you might be wnat to determine interface implementations without creating an instance of (you would need to know what class.)  This might be something that you want to add to the PluginManager, either as a method, or by adding that data to the getDefinitions() data (which should be cached)